README.md

```diff
@@ -30,3 +51,4 @@ Install this package:
 1. `git clone https://github.com/laws-africa/elk-docker.git`
 2. `cd elk-docker`
 5. `docker-compose up -d`
+Note that you will likely need to run docker-compose as sudo
```

Comment:
If you add your user to the `docker` group, you shouldn't need to.

Reply:
https://www.digitalocean.com/community/tutorials/how-to-install-and-use-docker-on-ubuntu-18-04#step-2-%E2%80%94-executing-the-docker-command-without-sudo-(optional)